server/src/main/java/org/opensearch/search/pipeline/AbstractProcessor.java

```diff
@@ -31,4 +31,9 @@ public String getTag() {
     public String getDescription() {
         return description;
     }
+
+    @Override
+    public boolean getIgnoreFailure() {
```

Comment:
The convention for `boolean` getters is to change the word `get` to `is`, so this would be `isIgnoreFailure()`. 

I wasn't sure, so I checked the [JavaBeans(TM) 1.01 specification](https://download.oracle.com/otndocs/jcp/7224-javabeans-1.01-fr-spec-oth-JSpec/), where section 8.3.2 says:

> In addition, for boolean properties, we allow a getter method to match the pattern:
`public boolean is<PropertyName>();`
This “is<PropertyName>” method may be provided instead of a “get<PropertyName>” method, or it may be provided in addition to a “get<PropertyName>” method.
In either case, if the “is<PropertyName>” method is present for a boolean property then we will
use the “is<PropertyName>” method to read the property value.
An example boolean property might be:
```
public boolean isMarsupial();
public void setMarsupial(boolean m);
```

So, it looks like it's a "common, but not required" convention. To me, the `get` method looks a little strange.

Reply:
changed to `isIgnoreFailure`